crates/matrix-sdk/src/client/mod.rs

```diff
@@ -588,6 +594,15 @@ impl Client {
     /// });
     /// client.remove_event_handler(handle);
     ///
+    /// // Registering custom event handler context:
+    /// struct MyContext {
+    ///     number: usize,
+    /// };
```

Comment:
```suggestion
    /// }
```

Reply:
whoops

Reply:
Can't push anymore to fix it, as auto-merge is enabled, but I suppose CI will fail? :D

Reply:
Oh you fixed already